### PR TITLE
update doc credits

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: shirly <AndreMouche@126.com>

### What is changed, added or deleted? (Required)

update doc for credits

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)

### What is the related PR or file link(s)?
no

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by AndreMouche** [comment]
@winkyao PTAL

File: credits.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+## SIG Leaders
```
Comment (lilin90) [inline credits.md:1]: 文档需要有一级标题和 metadata，请参考其他 `.md` 文件更新。

File: credits.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+| sig         | github    |
```
Comment (lilin90) [inline credits.md:3]:
```suggestion
| SIG name         | GitHub ID    |
```

File: credits.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+|[docs](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-docs)|[lilin90](https://github.com/lilin90)|
+|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[qw4990](https://github.com/qw4990)|
+|[tools](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-tools)|[kennytm](https://github.com/kennytm)|
+|[tiup](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-tiup)|[lonng](https://github.com/lonng)|
+|[web](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-web)|[wd0517](https://github.com/wd0517)|
+|[ddl](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl)|[zimulala](https://github.com/zimulala)|
+
+## Committers
+
+| sig         | github    |
```
Comment (lilin90) [inline credits.md:19]: ditto

File: credits.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
+## SIG Leaders
+
+| sig         | github    |
+|--:|--:|
```
Comment (lilin90) [inline credits.md:4]: 目前文档统一采用左对齐。
```suggestion
|:--|:--|
```

File: credits.md (hunk 1 above)
Comment (winkyao) [inline credits.md:1]: SIG Leaders 这个 section 我觉得不需要列，只需要列出专业的 role

File: credits.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
+---
+title: 荣誉列表
+summary: 了解 TiDB 截止发稿开发者贡献列表及角色
```
Comment (lilin90) [inline credits.md:3]: `截止发稿`不适合用在用户文档里。
```suggestion
summary: 了解 TiDB 社区贡献者列表及角色。
```

File: credits.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+category: contribute
+---
+
+TiDB 开发者社区以 SIG（Special Interest Groups） 为单位管理组织开发者。每个模块都有其固定的 SIG 负责新功能开发，性能优化，稳定性保障等。如果您想要成为 TiDB 的开发者，加入感兴趣的 SIG，与一线工程师面对面讨论，无疑是最好的方式。下文是截止 TiDB 4.0 GA，TiDB 社区开发者贡献列表及对应角色。
```
Comment (lilin90) [inline credits.md:7]:
```suggestion
TiDB 开发者社区以 SIG (Special Interest Group) 为单位管理组织开发者。每个模块都有其固定的 SIG 负责新功能开发、性能优化、稳定性保障等。如果你也想成为 TiDB 的开发者，加入感兴趣的 SIG、与一线工程师面对面讨论无疑是最好的方式。以下是截止 TiDB 4.0 GA，TiDB 社区贡献者列表及对应角色：
```

**Comment by winkyao** [comment]
@lilin90 PTAL again

File: credits.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
+---
+title: 荣誉列表
+summary: 了解 TiDB 社区贡献者列表及角色。
+category: contribute
+---
+
+TiDB 开发者社区以 SIG (Special Interest Group) 为单位管理组织开发者。每个模块都有其固定的 SIG 负责新功能开发、性能优化、稳定性保障等。如果你也想成为 TiDB 的开发者，加入感兴趣的 SIG、与一线工程师面对面讨论无疑是最好的方式。以下是截至 TiDB 4.0 GA，TiDB 社区贡献者列表及对应角色：
```
Comment (lilin90) [inline credits.md:7]:
```suggestion
TiDB 开发者社区以 SIG (Special Interest Group) 为单位管理组织开发者。每个模块都有其固定的 SIG 负责新功能开发、性能优化、稳定性保障等。如果你也想成为 TiDB 的开发者，加入感兴趣的 [SIG](https://github.com/pingcap/community/tree/master/special-interest-groups)、与一线工程师面对面讨论无疑是最好的方式。以下是截至 TiDB 4.0 GA，TiDB 社区贡献者列表及对应角色：
```

**Pull request merged**